src/solidity-frontend/solidity_convert.cpp

```diff
@@ -970,6 +970,19 @@ bool solidity_convertert::get_statement(
     new_expr = code_while;
     break;
   }
+
+  case SolidityGrammar::StatementT::Continue:
+  {
+    new_expr = code_continuet();
+    break;
+  }
+
+  case SolidityGrammar::StatementT::Break:
+  {
+    new_expr = code_breakt();
+    break;
+  }
+
```

Comment:
We can keep the StatementError, but we need to and a case here to handle the StatementError.